File: Telas/Pessoa.php
<?php
class Pessoa{
private $nomeCompleto;
    private $dataNascimento;
    private $telefone;
    private $email;

    public function getNome(){
        return $this->nomeCompleto;
    }
    public function setNome($nomeCompleto){
        $this->nomeCompleto = $nomeCompleto;
    }
    public function getDataNascimento(){
        return $this->dataNascimento;
    }
    public function setDataNascimento($dataNascimento){
        $this->dataNascimento = $dataNascimento;
    }
    public function getTelefone(){
        return $this->telefone;
    }
    public function setTelefone($telefone){
        $this->telefone = $telefone;
    }
    public function getEmail(){
        return $this->email;
    }
    public function setEmail($email){
        $this->email = $email;
    }

    //toString
    public function __toString(){
    return "Pessoa: Nome Completo: {$this->nomeCompleto} Data de nascimento: {$this->dataNascimento}  Telefone para contato: {$this->telefone} Email: {$this->email} ";
    }
}
?>

File: src/controller/JAVA/Pessoa.java
public abstract class pessoa {
	  private String nomeCompleto;
	  private String dataNascimento;
	  private String telefone;
	  private String email;
	  
	  public pessoa() {
		  
	  }
	  
	  public pessoa(String nomeCompleto, String dataNascimento, String telefone, String email) {
		  this.nomeCompleto = nomeCompleto;
		  this.dataNascimento = dataNascimento;
		  this.telefone = telefone;
		  this.email = email;
	  }

	public String getNomeCompleto() {
		return nomeCompleto;
	}

	public void setNomeCompleto(String nomeCompleto) {
		this.nomeCompleto = nomeCompleto;
	}

	public String getDataNascimento() {
		return dataNascimento;
	}

	public void setDataNascimento(String dataNascimento) {
		this.dataNascimento = dataNascimento;
	}

	public String getTelefone() {
		return telefone;
	}

	public void setTelefone(String telefone) {
		this.telefone = telefone;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	@Override
	public String toString() {
		return "Pessoa [nomeCompleto=" + nomeCompleto + ", dataNascimento=" + dataNascimento + ", telefone=" + telefone
				+ ", email=" + email + "]";
	}
	  

}
